tests/ResourceSyncingTest.php

```diff
@@ -877,4 +957,20 @@ public function getSyncedCreationAttributes(): array
     }
 }
 
+// override method in CentralUser class to return attribute names and default values
+class CentralUserWithAttributeNamesAndDefaultValues extends CentralUser {
+    public function getSyncedCreationAttributes(): array
+    {
+        // Sync name, email and password but provide default value for role
+        return
+            [
+                'global_id',
+                'name',
+                'password',
+                'email',
+                'role' => 'admin',
```

Comment:
It seems that this array is the same for the central model and the tenant model? According to what we discussed on Basecamp, we want to have tests which define this method differently in different (central/tenant) models.